src/components/NavBar/__tests__/NavBar.test.tsx

```diff
@@ -1,10 +1,11 @@
 import React from 'react';
 import { create } from 'react-test-renderer';
-import NavBar from '../NavBar';
+import NavBar from 'components/NavBar';
 
 jest.mock('@material-ui/core/AppBar', () => 'AppBar');
 jest.mock('@material-ui/core/Toolbar', () => 'Toolbar');
 jest.mock('@material-ui/core/IconButton', () => 'IconButton');
+jest.mock('buddy-app-schema', () => {});
```

Comment:
Since [PR-39](https://github.com/BuildIt-Poland/buddy-app/pull/39) is merged this mock is unnecessary